### DISCUSSION TITLE
added boost_uuid to BLIP-10

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#646] [Announcements - Changes Made]
This PR allows for a new property to be in the TLV record for identifying a boost across splits:  https://github.com/Podcastindex-org/podcast-namespace/pull/621

**Comment by brianoflondon** [comment]
Is it a bit much if we also added a `stream_uuid` for continuous sessions of streaming or is this too hard for apps to do?

I'm just working on my own API for sending, but I have a similar issue when I'm receiving streaming payments. On the receiving side for years I've batched up incoming payments and when 5 minutes passes without a new payment I consider a block of payments as complete.

Now on the sending side I'll still send payments immediately when an app hits my API, but from a back end point of view I'd like to record that they are a block of payments so later in a UI I can say to the user, you streamed 400 sats in 10 payments of 40 sats.

I think I can and will do this by creating a kind of app-name-hive-name-podcast-name unique ID for each streaming session.

I'm looking in my database and as far as I can see only Podverse, Castamatic and PodcastGuru have ever set the `uuid` for a payment and those seem to be unique.

Not sure if this is even something app devs would be bothered to add.

**Comment by brianoflondon** [comment]
I went ahead and added another pull request #647 for this.
